ts/features/idpay/onboarding/screens/MultiValuePrerequisitesScreen.tsx

```diff
@@ -98,9 +98,9 @@ const MultiValuePrerequisiteItemScreenContent = ({
           <H1>{I18n.t("idpay.onboarding.multiPrerequisites.header")}</H1>
           <VSpacer size={16} />
           <Body>{I18n.t("idpay.onboarding.multiPrerequisites.body")}</Body>
-          <Link>{I18n.t("idpay.onboarding.multiPrerequisites.link")}</Link>
+          <Label>{I18n.t("idpay.onboarding.multiPrerequisites.link")}</Label>
```

Comment:
Since that was a `Link` component, I think we have to pass the `asLink` prop

Reply:
@LeleDallas Unfortunately there is no `onPress` function attached, so I can't use the `asLink` prop to render it as a link. My guess is that it was already broken before my change. (Luckily `idPay` is still private, tough)

Reply:
Right... Was this intended to display a link in blue for cosmetic purposes?

Reply:
@LeleDallas Just for cosmetic purposes, I added the `asLink` prop with an empty function. Also added a `TODO` comment for the future developments. Here it is → https://github.com/pagopa/io-app/pull/6299/commits/12f2a9bc58e3ae968e75b44eed006a3ddf51cc3b